### PR TITLE
feat: breaking changes in geospatial-sdk [release]

### PR DESCRIPTION
**Pull request by Sollimann**
[SBS-2782] - Update JS SDK after breaking changes in the Geospatial SDK

On another note, Travis seems to hang a lot today. The jobs remain queued for a really long time.

[SBS-2782]: https://cognitedata.atlassian.net/browse/SBS-2782

### CODE REVIEW

**Review by johnberntsen — APPROVED** [review]
LGTM

**Comment by Sollimann** [comment]
Will the version automagically bump to 0.1.1 now? It didn't bump the version in the last PR. Is it the **feat** keyword or the **[release]** keyword that bumps the version? @polomani

**Comment by polomani** [comment]
`fix:` automatically bumps patch, `feat:` bumps minor
but if you are on 0.x then every change is considered to be breaking, so I am not completely sure what you'll get now, most likely 0.2.0.
`feat: fix` is really confusing, you better stick to one. remember this goes directly to changelog


`[release]` just triggers a release

**Comment by Sollimann** [comment]
thanks, Can I get an approval on the PR @polomani @hhenriques ? Need one from core-api or devx-externals for merging.

**Review by hhenriques — APPROVED** [review]
I didn't actually review the changes since I don't have context. We should look into making your team code owners of the wells package.

**Comment by polomani** [comment]
@hhenriques it is, it's just the `yarn.lock` and `package.json` that prevents them from merging.
and I don't see a nice solution to this..

**Pull request merged**
